lsp-haskell.el

```diff
@@ -238,7 +310,23 @@ and `lsp-haskell-server-args' and `lsp-haskell-server-wrapper-function'."
                                 ("haskell.liquidOn" lsp-haskell-liquid-on t)
```

Comment:
Is the liquid haskell support also going to become a plugin?

Reply:
There are a whole lot of legacy configuration items we brought over from HIE, some of which do things, some don't. 

We have kept them so the parser doesn't blow up on the hls side if they get sent.

And we probably need a better deprecation story. It *is* important to report errors for malformed config, but it can become annoying.